app/assets/stylesheets/_orders.scss

```diff
@@ -0,0 +1,16 @@
+#order-title{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space